scst-store/configuration/use-load-balancer.hbs.md

```diff
@@ -0,0 +1,39 @@
+# Using LoadBalancer
+
+To use a `LoadBalancer` configuration, you must find the external IP address of the `metadata-store-app` service by using kubectl.
```

Comment:
This part doesn't actually describe what is being achieved in the code block below.

Reply:
I changed this sentence, do you think this makes more sense?